src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -70,6 +72,16 @@ export class AddressForm extends React.Component<
     })
   }
 
+  getError = (key: keyof Address): string => {
+    return (
+      (this.props.touched &&
+        this.props.touched[key] &&
+        this.props.errors &&
+        this.props.errors[key]) ||
+      ""
```

Comment:
@erikdstock For this, it looks like we're returning an empty string instead of `null` or `undefined` here. Should we change that? Or is it fine as-is?